src/app/features/increase-fee-drawer/components/increase-fee-field.tsx

```diff
@@ -44,26 +44,21 @@ export function IncreaseFeeField(props: IncreaseFeeFieldProps): React.JSX.Elemen
           showReset={showResetMultiplier}
           onSelectMultiplier={onSelectMultiplier}
         />
-        <Flex>
-          <styled.label display="block" textStyle="body.02" mb="space.02" htmlFor="fee">
```

Comment:
I think we need to keep the label here for accessibility? Shouldn't we just hide it or wrap the input?

Reply:
yeah, have hidden it